src/site/apt/index.apt

```diff
@@ -8,7 +8,7 @@ Maven Jenkins Plugin
   This maven2 plugin provides various goals for developing Jenkins plugins.
 
 
-* Goals Overview
+* Common Goals Overview
```

Comment:
Yes - but there is nothing on this page so the anchor is mostly pointless

ref: https://jenkinsci.github.io/maven-hpi-plugin/